CU-04 Filter events

santiborge97 opened the pull request.

santiborge97 committed:
[feat] Nuevo form para filtrar

santiborge97 committed:
[feat] Algunos avances

santiborge97 committed:
[feat] Filtro no funciona

santiborge97 committed:
[fix] Filter search done

angbongon commented:
Corrige los code smells que no son de CSS o HTML

angbongon committed:
Merge branch 'dev' into feat/be/CU-04-Filtrar-eventos

santiborge97 committed:
[fix] Sin paginacion

santiborge97 committed:
[fix]Pagina pero sin context

santiborge97 committed:
[fix] Valida pero pagina regular

Commit:
[fix][refactor] CU-04 con paginación y categorias

angbongon commented:
Muchos de los bugs y code smells pertenecen a templates que ya no se usan, otros son por tema de código comentado que se dejará así para el sprint 3

angbongon committed:
Merge branch 'dev' into feat/be/CU-04-Filtrar-eventos

santiborge97 committed:
[fix] query set puesto

santiborge97 committed:
Merge branch 'feat/be/CU-04-Filtrar-eventos' of https://github.com/eventshow/eventshow into feat/be/CU-04-Filtrar-eventos

Commit:
[hack] Quitado categorias en búsqueda

Debido a que en el nearby se devuelve una lista no podemos hacer una
subquery de esta para obtener un group by

Commit:
Merge branch 'feat/be/CU-04-Filtrar-eventos' of github.com:eventshow/eventshow into feat/be/CU-04-Filtrar-eventos

angelcalzadollamas reviewed (changes requested):
En precio mínimo y precio máximo si pongo: 2. ó .0 falla el pattern

angbongon committed:
[fix] Calzado tus muertos

Se permite en las urls introducir valores para los precios sin parte entera

angelcalzadollamas reviewed (changes requested):
Prueba a poner 0. Te fallará

angbongon commented:
> Prueba a poner 0. Te fallará

No estoy viendo ese error

angbongon committed:
Update urls.py

angelcalzadollamas reviewed (approved):
Todo correcto, buen trabajo
